### ISSUE TITLE
New tags for PVE vs PVP

### ISSUE THREAD

**Issue by 13DavidG** [DestinyItemManager/DIM#3050]
This is a simple suggestion. With the arrival of the random rolls and the abundance of weapons/armors, it is getting more and more confusing and difficult to keep track of our "favorite" weapons. Personally, I use tags just for the symbols and give them my own meaning, but it is not intuitive. However, I feel like having a PVE and a PVP tag to differentiate weapons would help a lot to organize our inventory, especially with duplicate weapons with different rolls and such. 
Basically, we can differentiate the ratings and it would be great to be able to differentiate tags or have new ones labeled PVP and PVE

**Comment by kyleshay** [comment]
We've thought about this, and have historically held off as it is possible to use the notes section to help for this deeper level of customization.

For example if you write "PVP" in your notes section, and use the search filter `notes:PVP`, every item that has "PVP" in the notes section will return. 

Being said, I'm open to the idea of allowing users to any number of custom tags. The framework is setup for it, but we haven't fully thought it out yet.

**Comment by patchm00** [comment]
Added an issue for the above.  I tried the suggestion of adding NOTES and the next morning all my notes were gone

**Comment by inexorableAce** [comment]
my main issue is that tags currently has a very specific and tidy purpose. Currently the tags have minimal overlap this is again because currently they all have the same basic purpose of what you want to do with an item where "favorite" trumps "keep". 

Adding "pve" and "pvp" changes that dynamic. There are things that could be 3 tags easily like sleeper for instance, its great in pvp, pve and its a favorite. It also opens up questions like: what about gambit? Why is that not a tag but pvp and pve are? 

I just see no compelling reason that notes can't serve this purpose of custom tagging.

**Comment by bhollis** [comment]
We chatted about this a bit, and I think @inexorableAce put it well - we don't necessarily want to expand to an arbitrarily large number of tags, and the tags today are pretty "neutral". I'd suggest using notes for this, as well.
